src/renderer/services/midgard/pools.ts

```diff
@@ -254,8 +202,8 @@ const createPoolsService = (
       liveData.chain(
         O.fold(
           // TODO (@thatStrangeGuy | @veado) Add i18n
-          (): PoolOverviewDetailsLD => Rx.of(RD.failure(new Error('No pools available'))),
-          (poolsDetails): PoolOverviewDetailsLD => Rx.of(RD.success(poolsDetails))
+          (): PoolDetailsLD => Rx.of(RD.failure(new Error('No pools available'))),
```

Comment:
i18n ^

Reply:
Yup, it's still a TODO (see line `204`) - we can ignore it for now.